openai/openai.go

```diff
@@ -474,7 +476,7 @@ func FromChatRequest(r ChatCompletionRequest) (*api.ChatRequest, error) {
 					if !ok {
 						return nil, errors.New("invalid message format")
 					}
-					messages = append(messages, api.Message{Role: msg.Role, Content: text})
```

Comment:
So seems like we were keeping user messages separate in this case where you could go user -> user -> image. Wouldn't we want user -> user+image instead? where image is added to the last user message?